EIPS/eip-7610.md

```diff
@@ -0,0 +1,47 @@
+---
+eip: 7610
+title: Revert creation in case of non-empty storage
+description: Revert contract creation if address already has the non-empty storage
```

Comment:
Avoid restating the title in the description. Maybe include why this is important to do.

Reply:
https://eips.ethereum.org/EIPS/eip-684 uses the same terminology.